samples/snippets/src/main/java/com/example/spanner/QueryLab.java

```diff
@@ -0,0 +1,244 @@
+        rowCount++;
+      }
+      // Writing all the rows in different tables.
+      dbClient.write(mutationsPerson);
+      dbClient.write(mutationsContact);
+      dbClient.write(mutationsAddress);
+
+    }
+  }
+
+  /**
+   * Helper method to generate the test data for Person table.
+   *
+   * @param personId unique Id value.
+   * @return @{@link Mutation} object
+   */
+  private Mutation getPersonTableDummyValue(String personId) {
+    return Mutation.newInsertBuilder("Person")
+        .set("person_id")
+        .to(personId)
+        .set("name")
+        .to(UUID.randomUUID().toString())
+        .set("age")
+        .to(new Random().nextInt(100 - 10 + 1) + 10)
+        .set("timestamp")
+        .to(Value.COMMIT_TIMESTAMP)
+        .build();
+  }
+
+  /**
+   * Helper method to generate the test data for Contact table.
+   *
+   * @param contactId unique Id value for contactId.
+   * @return @{@link Mutation} object
+   */
+  private Mutation getContactTableDummyValue(String contactId) {
+    return Mutation.newInsertBuilder("Contact")
+        .set("contact_id")
+        .to(contactId)
+        .set("email")
+        .to(
+            String.format(
+                "test%s@%stest.com", new Random().nextInt(100), new Random().nextInt(1000)))
+        .set("phone")
+        .to(generateRandomPhoneNumber())
+        .set("timestamp")
+        .to(Value.COMMIT_TIMESTAMP)
+        .build();
+  }
+
+  /**
+   * Helper method to generate the test data for Address table.
+   *
+   * @param addressId unique Id value for address table.
+   * @param personId unique id value of the person table which is also save in person table.
+   * @return @{@link Mutation} object
+   */
+  private Mutation getAddressTableDummyValue(String addressId, String personId) {
+    return Mutation.newInsertBuilder("Address")
+        .set("address_id")
+        .to(addressId)
+        .set("person_id")
+        .to(personId)
+        .set("address")
+        .to(UUID.fromString("123e4567-e89b-12d3-a456-426655440000").toString())
+        .set("zip")
+        .to(new Random().nextInt(99999) + 10000)
+        .set("city")
+        .to("test" + generateRandomState())
+        .set("state")
+        .to(generateRandomState())
+        .set("timestamp")
+        .to(Value.COMMIT_TIMESTAMP)
+        .build();
+  }
+
+  /** Helper method to generate the random phone number. */
+  private String generateRandomPhoneNumber() {
+    Random random = new Random();
+
+    // Generating area code (3 digits)
+    int areaCode = random.nextInt(999); // Random area code between 200 and 999
+
+    // Generating exchange code (3 digits)
+    int exchangeCode = random.nextInt(999); // Random exchange code between 100 and 999
+
+    // Generating subscriber number (4 digits)
+    int subscriberNumber = random.nextInt(9999); // Random subscriber number between 1000 and 9999
+
+    return String.format("(%03d) %03d-%04d", areaCode, exchangeCode, subscriberNumber);
```

Comment:
Can we remove the variables above and just call random directly in here? This function can probably be one line.